docs/user/README.md

```diff
@@ -1,6 +1,8 @@
-+++
-draft = "True"
-+++
+---
+aliases:
+- /docs/agent/latest/README/
+draft: "True"
```

Comment:
As far as I can tell, this is working on the website. https://grafana.com/docs/agent/latest/README/ is a 404. I was expecting this to be a boolean value though.